app/helpers/nav_helper.rb

```diff
@@ -84,6 +84,12 @@ def subnav_items(main = nil)
             path: households_path,
             permitted: policy(Household.new(community: current_community)).index?,
             icon: "home"
+          }, {
+            name: :roles,
+            parent: :people,
+            path: roles_path,
+            permitted: policy(User).index?,
+            icon: "wrench"
```

Comment:
Can you pick a different icon? A wrench doesn't seem super appropriate.

https://fontawesome.com/v4.7.0/icons/#chart